sdmetrics/reports/single_table/diagnostic_report.py

```diff
@@ -178,6 +181,22 @@ def get_visualization(self, property_name):
             plotly.graph_objects._figure.Figure
                 The visualization for the requested property.
         """
+        score_breakdowns = {
+            metric.__name__: self._metric_results[metric.__name__]
+            for metric in self.METRICS.get(property_name, [])
+        }
+
+        if property_name == 'Coverage':
+            fig = get_column_coverage_plot(score_breakdowns, self._property_scores[property_name])
+
+        elif property_name == 'Boundaries':
+            fig = get_column_boundaries_plot(
+                score_breakdowns, self._property_scores[property_name])
+
+        elif property_name == 'Synthesis':
+            fig = get_synthesis_plot(score_breakdowns.get('NewRowSynthesis', {}))
+
```

Comment:
I think we should use `property_name.lower() == 'coverage'` and ensure that even if the user doesn't write 'Coverage' with capital `c` this is being properly shown (same for the other cases).

Reply:
Hm, I'll run this by Neha and if we want to make this change, it will be a larger change (addressed by a separate PR) since we will need to be consistent across all reports and APIs.